app/models/diaper_drive.rb

```diff
@@ -56,6 +58,10 @@ def donation_source_view
     "#{name} (diaper drive)"
   end
 
+  def total_money_raised
+    donations.sum(&:money_raised)
+  end
```

Comment:
This _may_ result in an N+1 if the donations are not eager-loaded. 

You could maybe add it as an association extension to the `has_many :donations` (line 28) or you could do it in the Service object to ensure that it's always eager loaded. 

If neither one of those, throw a TODO on this method (or create an issue, I'm not sure what the team is doing now) to address it later

Reply:
I think just removing the '&' will do it!